### ISSUE TITLE
Whoops!, it looks like the server returned an error. when using embeded view

### ISSUE THREAD

**Issue by GeekLabsUK** [Attendize/Attendize#964]
**Describe the bug**

I used the html embed code and the tickets are displaying fine but when you try to order a ticket you get a 'Whoops!, it looks like the server returned an error. Please try again, or contact the webmaster if the problem persists.' error.

i can order tickets directly from the event page but not from within the embeded widget.

you can see live at www.stourbridgescaremaze.co.uk 

**To Reproduce**

1. Go to : 
www.stourbridgescaremaze.co.uk  and scroll down or hit 'book now' select a date and tickets for the date are displayed. This is simply hiding or displaying a div containing the iframe. try to select some tickets and hit register.

try directly on the event page - https://www.4wardtickets.co.uk/e/2/friday-8th-october and it works fine

**Comment by GeekLabsUK** [comment]
I tried using the main event page within the iframe and this also didn't work. It seems to be it has something to do with iframes? Embedding tickets on our own websites is a pretty important feature that needs to work.

**Comment by samdb** [comment]
This is a legacy feature. Most browsers will no longer allow you to submit forms within an iframe now.

The only way I can think of to get this to work would be if the submit button triggers a redirect to the standalone page.

**Comment by GeekLabsUK** [comment]
eventbrite allow you to embed tickets in an iframe so it can be done. Its an important feature that event organisers want.

**Comment by samdb** [comment]
Yes, and they break out of the iframe as soon as you click add to cart.

**Comment by quentincaffeino** [comment]
@samdb can I have a link to that info please?

I tried the embedded form with devtools and response from the server says `CSRF token mismatch`. If we look at the cookie sent with the iframe content it says that because SameSite option isn't set chrome defaults it to Lax which means cookie isn't set because websites operate on different domains.
![image](https://user-images.githubusercontent.com/2855777/130050968-99d6bb9e-3ae9-46fe-adfb-306a14bc6feb.png)

This breaks laravel sessions (it will 100% break next form with credentials since it depends on session object) and I *think* it breaks csrf checks which I *think* are also tied to user session.

I don't really know the solution to that but you can try setting `SESSION_SECURE_COOKIE` env variable to true and inside `config/session.php` setting `same_site` to `'None'`

**Comment by GeekLabsUK** [comment]
> Yes, and they break out of the iframe as soon as you click add to cart.

No they don't. Ive been using Eventbrite for years in that way and the user stays within the iframe. However it wouldn't bother me if it did break out of the iframe. The only reason its an important feature is most event organisers want there ticketing integrated with their website and all of the big boy ticketing software allow for it.

So moving forward what options are available. Its obviously doable as it being done by other companies.  How van we get it to break out of the iframe?

**Comment by GeekLabsUK** [comment]
> I don't really know the solution to that but you can try setting `SESSION_SECURE_COOKIE` env variable to true and inside `config/session.php` setting `same_site` to `'None'`

Doing this works, well allows us to break out of the iframe so at least my tickets can be purchased. thanks for your help.

**Comment by el3ashe2** [comment]
> > I don't really know the solution to that but you can try setting `SESSION_SECURE_COOKIE` env variable to true and inside `config/session.php` setting `same_site` to `'None'`
> 
> Doing this works, well allows us to break out of the iframe so at least my tickets can be purchased. thanks for your help.

didn't work for me and error still here
